Automattic/wp-calypso
# Reorganize, simplify, deduplicate typings

#### Changes proposed in this Pull Request

* Organizes mapped-types into aptly named `mapped-types` 🙂 
* Provide a header describing contents of `mapped-types`.
* Add reusable mapped-types `DispatchFromMap ` and `SelectFromMap` which are reused in our stores.

#### Testing instructions

* select and dispatch on the stores continue to be well typed (no change)
* App builds without issue (reorganization was successful

These are all type level changes. If the app builds, it should be safe.
